### DISCUSSION TITLE
Will react-native-unistyles V3.0 also add Plugin support?

### DISCUSSION THREAD

**Discussion by dppo** [jpudysz/react-native-unistyles#361] [Q&A]
I just found out that V3.0.0-beta has been released, but the Plugin feature has been deleted. Will you consider adding Plugin support in the future?
or are there other features that can support global interception and modification of Styles?

**Comment by jpudysz** [comment]
Hey man, at this point no. 

Anything what was possible with the plugins can be done statically with JS. 

I'm just wondering if I can parallelize code parsing or push it to different threads. If so, I can create some kind of worklet functions. 

Stay tuned!
